rollup.config.js

```diff
@@ -2,18 +2,31 @@ import path from 'path'
 import babel from '@rollup/plugin-babel'
 import resolve from '@rollup/plugin-node-resolve'
 import typescript from '@rollup/plugin-typescript'
+import esbuild from 'rollup-plugin-esbuild'
 import { sizeSnapshot } from 'rollup-plugin-size-snapshot'
 
 const createBabelConfig = require('./babel.config')
-
 const { root } = path.parse(process.cwd())
-const external = (id) => !id.startsWith('.') && !id.startsWith(root)
+const external = (id) =>
+  (!id.startsWith('.') && !id.startsWith(root)) ||
+  id.startsWith('@babel/runtime')
+
 const extensions = ['.js', '.ts', '.tsx']
 const getBabelOptions = (targets) => ({
   ...createBabelConfig({ env: (env) => env === 'build' }, targets),
   extensions,
+  comments: false,
+  babelHelpers: 'runtime',
```

Comment:
what's the reason we need this? we don't use any async/await.

Reply:
Without it babel helpers would be included by default as the `bundled` setting (vs runtime). See here https://github.com/rollup/plugins/tree/master/packages/babel#babelhelpers. it gives this message: 

> It is recommended to configure this option explicitly (even if with its default value) so an informed decision is taken on how those babel helpers are inserted into the code.

I set it explicitly, and set it to `runtime` as I think that is the smallest footprint. I don't think helpers are only for async functions, but I might be wrong!

Reply:
Perhaps we could disable helpers all together?

Reply:
I might be misunderstanding. Let me play with it a bit.

Reply:
Okay, I misunderstood something. so the default is `bundled` and it's not recommended.

So, let's say we use `runtime` as recommended. My questions are:

- Can you eliminate it from dependenceis, as zustand doesn't have it?
  - My feeling is "no", in this case, we should put it in peerDependencies with optional meta, for all zustand/jotai/valtio
- Don't we need `@babel/plugin-transform-runtime` here as zustand has it?

Reply:
I did some more experiments. Here's the cjs bundle size (not minified.)

| | `'runtime'` | `'bundled'` | diff |
|---|---|---|---|
|index.js|22123|22349|+226|
|utils.js|13887|12907|-908|
|devtools.js|2395|2395|0|
|immer.js|1964|1964|0|
|optics.js|1868|1868|0|
|query.js|3291|3291|0|
|xstate.js|3074|3062|-12|

So, given this resulting size, we could actually go with`bundled`, which is far less troublesome.

Reply:
(btw, would it be possible for you to DM me in Discord? Just got a bit tired going back and forth in these three repos.)